docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -100,128 +117,212 @@ In the case of errors, include human-readable error messages in the response pay
 }
 ```
 
-## Developing the app
+<!-- type: tab-end  -->
 
-To use the Shipping Provider API to provide shipping quotes, shipping providers must build a BigCommerce [single-click app](/api-docs/apps/guide/types#single-click). 
+## Develop the app
 
-Using a BigCommerce app enables shipping providers to promote their solution in the BigCommerce apps marketplace, request merchant authorization of API scopes during app install, and enable configuration of shipping provider settings and/or order fulfillment via an iFrame in the BigCommerce control panel.
+To use the Shipping Provider API to provide shipping quotes, you must build a BigCommerce [single-click app](/api-docs/apps/guide/types#single-click). For more info on how to get started, see the [BigCommerce Apps Quick Start](/api-docs/partner/getting-started/app-development/tutorials/quick-start) article.
 
-For more information, see our [Introduction to Building Apps](/api-docs/apps/guide/intro).
+Building a BigCommerce app allows you to create [app API credentials](/api-docs/getting-started/authentication/authenticating-bigcommerce-apis#app-api-credentials) and promote your solution in the BigCommerce app marketplace. Having a BigCommerce app allows merchants to configure shipping provider settings and/or order fulfillment via an iFrame in the BigCommerce control panel.
 
-## Control Panel installation workflow
+For more information, see our [Introduction to Building Apps](/api-docs/apps/guide/intro) article.
 
-During the app setup, if you configure the Check Connection Options URL for the carrier, an attempt to connect the carrier via the Shipping Manager UI or the Connect Carrier API causes a request to be made to that URL with the provided options. The resource should respond by indicating if the credentials are valid and explain what is wrong. If you did not configure this URL, this check is not required and the credentials are assumed valid as long as they pass type checks.
+### Configuration fields

```

Comment:
Not sure if this section is more useful in the "Before Development" section, or if some of the sections in "Before Development" should go in "Develop the App"

Reply:
I have combined the sections into one "Developing the App" section